frontend/server/src/main/java/org/pytorch/serve/job/RestJob.java

```diff
@@ -163,8 +163,8 @@ private void responseInference(
             "{}",
             new Metric(
                     "RequestPriority",
-                    String.valueOf(this.getPriority()),
-                    "int",
+                    this.getPriority(),
+                    "category",
```

Comment:
Should this be a metric? It logs more like a log entry to me. (as it is not something like a count, gauge, histogram). I would say, we should rather add the priority as an additional label to other metrics, such as `QueueTime`  above.

Reply:
Yes, ideally I would add this as a label to another metric. However, I think `torchserve`'s `Metric` does not support adding arbitrary labels, and I don't want to refactor the metrics system in this PR.

I think the feasible options here are to keep as it currently is (a request-level logged metric), make it a prometheus `Counter` metric (a cumulative counter for requests of each priority in the metrics api), or drop it entirely. Of these, which do you think makes the most sense? Whichever we choose, I think we can implement it together with the other queue metrics in #5.

Reply:
I think maybe a counter would be the best then - but you are right, lets do the metrics with #5.